runtime/doc/api.txt

```diff
@@ -463,7 +477,15 @@ We can get all marks in a buffer for our namespace (or by a range): >
     => [[1, 0, 2]]
 
 Deleting all text surrounding an extmark does not remove the extmark. To
-remove an extmark use |nvim_buf_del_extmark()|.
+remove an extmark use |nvim_buf_del_extmark()|. Deleting "x" in our example
+would result in the following.
+
+      0 12345678
+    0 e|ample..
+       ^ This is the extmark position
+
+    Note: An extmark "gravity" meaning the direction it will be shifted after a
+    text edit can be adjusted see |nvim_buf_set_extmark()| for more information.
```

Comment:
This sounds a bit weird, maybe:
```suggestion
    Note: An extmark "gravity" is the direction it will be shifted after a
    text edit. See |nvim_buf_set_extmark()| for more information.
```